resources/definitions/fdmprinter.def.json

```diff
@@ -4744,9 +4790,8 @@
                     "minimum_value_warning": "raft_interface_line_width",
                     "maximum_value_warning": "20",
                     "enabled": "resolveOrValue('adhesion_type') == 'raft'",
-                    "limit_to_extruder": "adhesion_extruder_nr",
                     "settable_per_mesh": false,
-                    "settable_per_extruder": true
+                    "settable_per_extruder": false
```

Comment:
These need to be settable per extruder and limited to the adhesion extruder, otherwise a quality, variant or material profile can't determine this line width any more.